### DISCUSSION TITLE
#46 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#25] [General]
Agenda:

Community updates
Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by luckyyang** [comment]
### Rebase

[活动](https://github.com/rebase-network/event-pool/issues)
![2022-08-22 at 17 50](https://user-images.githubusercontent.com/1963646/185893211-9eb8f7a8-10c0-4ad3-9774-3b36fe765df3.png)

[招聘](https://github.com/rebase-network/who-is-hiring/issues)
![2022-08-22 at 17 52](https://user-images.githubusercontent.com/1963646/185893454-6ed87c2f-30c5-4446-b55a-552f9bc678d2.png)

[Web3 daily](https://rebase.network/posts/category/web3daily)
更新到 946 期

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

6个PR #815 ～ #821

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会

- 《YFI 源码解读》 by Joly
  - https://www.youtube.com/watch?v=wPhNCijEMec
  - https://www.youtube.com/watch?v=y4vAxSsKmRY

- 《Chainlink 原理剖析》by Frank
  - https://www.youtube.com/watch?v=dtcUo24wug4
  - https://www.youtube.com/watch?v=2pkQvoct4JE
  
- 《Mitigating Bridge Attacks with On Chain Rate Limiting Contracts》by Qizhou
  - https://www.youtube.com/watch?v=Ob6IpkvDACo

## 翻译小组

- MIT公开课正式发布第22集
- 第23 审稿中

**Comment by Riely007** [comment]
### 以太坊翻译小组

- advanced page 翻译进行中

**Comment by bambalados** [comment]
**CTN**

- 开始了 CTN_EN Twitter 号 [link](https://twitter.com/CTNenglish)
- Planning collab with [Tampa Bay DAO](https://tampabaydao.com/)
- Planning CTN 17 (prob in 2-3 weeks)
- 讨论发展规划， 欢迎新小伙伴

**Comment by stephanieeee-ho** [comment]
### ECN
1. 常规翻译、七日谈和 E 群誌
2. 大理以太坊主题营地策划筹备到抽毯

**Comment by Stamp9** [comment]
## 原语里弄
大理活动（未举行）

**Comment by Emilyxiannv** [comment]
DAOSquare
内容输出：
-翻译文章
DAO社区及其健康评价：RnDAO研究报告 -> https://dao2.muse.lol/article/43
初创公司：你应该成为一个去中心化的自治组织吗？-> https://dao2.muse.lol/article/41
可编程组织和 DAO 可组合性-> https://dao2.muse.lol/article/39
-Twitter Space
联合Buidler DAO 一起做了一场分享 “Investment in Web3” -> https://twitter.com/i/spaces/1yNGaYbDzaRGj

**Comment by 0x68616f79756e** [comment]
ETHPlanet：
1.支持旧金山 大型活动 Ethereum a new era 活动协调
2.筹备 ETH tech weeks
